pkg/driverbuilder/builder/archlinux.go

```diff
@@ -23,7 +23,6 @@ type archlinux struct {
 type archlinuxTemplateData struct {
 	commonTemplateData
 	KernelDownloadURL string
-	GCCVersion        string
```

Comment:
GCCVersion is not embed in `commonTemplateData` and automatically selected by a not-so-smart algorithm, when GCCVersionRequestor interface is not implemented by a builder.